.github/workflows/release.yml

```
@@ -28,16 +35,17 @@ jobs:
           echo "" >> release-note.md
           echo "### Translations" >> release-note.md
           echo "" >> release-note.md
-          version=${GITHUB_REF_NAME#v}
-          major_version=${version%%.*}
+          tag=${{ inputs.tag }}
+          version=${tag#v}
+          major_version=${version%%.*}inputs.tag
           version_hyphen=$(echo ${version} | tr . -)
           echo "  * [Japanese](https://groonga.org/ja/docs/news/${major_version}.html#release-${version_hyphen})" >> release-note.md
           title="$(head -n1 release-note.md | sed -e 's/^## //')"
           tail -n +2 release-note.md > release-note-without-version.md
-          gh release create ${GITHUB_REF_NAME} \
+          gh release create ${{ inputs.tag }} \
```

```suggestion
          gh release create "${tag}" \
```

`${tag}` を設定したのでそれを使うほうがわかりやすいかも。